src/cljs/webchange/ui_framework/layout/right_menu/object_tree_menu/views.cljs

```diff
@@ -0,0 +1,45 @@
+(ns webchange.ui-framework.layout.right-menu.object-tree-menu.views
+  (:require
+    [re-frame.core :as re-frame]
+    [webchange.editor-v2.activity-form.common.objects-tree.state :as state]
+    [webchange.ui-framework.components.index :refer [icon]]))
+
+(defn- objects-tree-item
+  [{:keys [alias name]}]
+  (let [visible? @(re-frame/subscribe [::state/visible? name])
+        show-remove-button? @(re-frame/subscribe [::state/show-remove-button? name])
+        handle-visibility-click #(re-frame/dispatch [::state/set-object-visibility name (not visible?)])
+        handle-remove-click #(re-frame/dispatch [::state/remove-object name])]
+    [:div
+     [:div.float-left
```

Comment:
I don't like this approach.. Blocks should have meaningful names (class names or ids)  like 'object-name', 'object-actions', etc. And in .scss file we define how this named block should looks like. And reading html we should understand what is function of this block, not style.
Otherwise using '.float-left', '.margin-right' and others like this class names we just duplicate css in html.